src/__tests__/scss/__snapshots__/styles-test.js.snap

```diff
@@ -3594,7 +3594,7 @@ body {
 }
 
 .security--card__link {
-  transition-duration: 0.2s;
+  transition-duration: 240ms;
```

Comment:
Are there no tokens for the durations?

Reply:
> $transition-duration: 0.2s;
$transition-duration: $moderate-02;

Reply:
There is! This is a snapshot of the changes to the CSS output.